src/main/java/frc/robot/robotcontainers/CompRobotContainer.java

```diff
@@ -20,6 +20,13 @@
  */
 public class CompRobotContainer extends RobotContainer {
 
+  public enum RobotState {
```

Comment:
WPIlib already has a class of RobotState for driver station's state.
Suggested to use a different name, e.g., RobotGamePieceState.